GoogleJavaFormat fails with NPE

## What version of OpenRewrite are you using?

- org.openrewrite:rewrite-core:8.66.0

## What is the smallest, simplest way to reproduce the problem?

We're using `org.openrewrite.java.GoogleJavaFormat` and when upgrading to 8.66.0 it fails in our tests with:

```
java.lang.AssertionError: Failed to run recipe at Cursor{ClassDeclaration->CompilationUnit}
	at org.openrewrite.test.RewriteTest.lambda$defaultExecutionContext$14(RewriteTest.java:648)
	at org.openrewrite.scheduling.RecipeRunCycle.handleError(RecipeRunCycle.java:291)
	at org.openrewrite.scheduling.RecipeRunCycle.lambda$editSources$8(RecipeRunCycle.java:230)
	at org.openrewrite.scheduling.RecipeStack.reduce(RecipeStack.java:60)
	at org.openrewrite.scheduling.RecipeRunCycle.lambda$editSources$9(RecipeRunCycle.java:179)
	at org.openrewrite.internal.InMemoryLargeSourceSet.lambda$edit$0(InMemoryLargeSourceSet.java:86)
	at org.openrewrite.internal.ListUtils.map(ListUtils.java:245)
	at org.openrewrite.internal.ListUtils.map(ListUtils.java:269)
	at org.openrewrite.internal.InMemoryLargeSourceSet.edit(InMemoryLargeSourceSet.java:85)
	at org.openrewrite.scheduling.RecipeRunCycle.editSources(RecipeRunCycle.java:177)
	at org.openrewrite.RecipeScheduler.runRecipeCycles(RecipeScheduler.java:84)
	at org.openrewrite.RecipeScheduler.scheduleRun(RecipeScheduler.java:41)
	at org.openrewrite.Recipe.run(Recipe.java:436)
	at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:386)
	at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:132)
	at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:127)
	at com.spotify.bazel.rewrite.recipes.StaticImportsTest.staticImports(StaticImportsTest.java:21)
Caused by: org.openrewrite.internal.RecipeRunException: java.lang.NullPointerException: Cannot invoke "java.lang.Boolean.booleanValue()" because the return value of "org.openrewrite.java.style.SpacesStyle$BeforeLeftBrace.getClassLeftBrace()" is null
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:278)
	at org.openrewrite.java.format.SpacesVisitor.visit(SpacesVisitor.java:1130)
	at org.openrewrite.java.format.SpacesVisitor.visit(SpacesVisitor.java:31)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:154)
	at org.openrewrite.TreeVisitor.visitNonNull(TreeVisitor.java:173)
	at org.openrewrite.java.format.AutoFormatVisitor.visit(AutoFormatVisitor.java:87)
	at org.openrewrite.java.format.AutoFormatVisitor.visit(AutoFormatVisitor.java:43)
	at org.openrewrite.java.JavaVisitor.autoFormat(JavaVisitor.java:95)
	at org.openrewrite.java.AddImport.lambda$preVisit$3(AddImport.java:189)
	at org.openrewrite.internal.ListUtils.mapFirst(ListUtils.java:166)
	at org.openrewrite.internal.ListUtils.mapFirst(ListUtils.java:183)
	at org.openrewrite.java.AddImport.preVisit(AddImport.java:188)
	at org.openrewrite.java.AddImport.preVisit(AddImport.java:59)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:239)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:265)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:154)
	at org.openrewrite.Preconditions$Check.visit(Preconditions.java:159)
	at org.openrewrite.Preconditions$Check.visit(Preconditions.java:130)
	at org.openrewrite.scheduling.RecipeRunCycle.lambda$editSources$7(RecipeRunCycle.java:210)
	at org.openrewrite.table.RecipeRunStats$PhaseTimer.recordTimed(RecipeRunStats.java:142)
	at org.openrewrite.table.RecipeRunStats$RecipeTimers.recordEdit(RecipeRunStats.java:130)
	at org.openrewrite.table.RecipeRunStats.recordEdit(RecipeRunStats.java:58)
	at org.openrewrite.scheduling.RecipeRunCycle.lambda$editSources$8(RecipeRunCycle.java:206)
	at org.openrewrite.scheduling.RecipeStack.reduce(RecipeStack.java:60)
	at org.openrewrite.scheduling.RecipeRunCycle.lambda$editSources$9(RecipeRunCycle.java:179)
	at org.openrewrite.internal.InMemoryLargeSourceSet.lambda$edit$0(InMemoryLargeSourceSet.java:86)
	at org.openrewrite.internal.ListUtils.map(ListUtils.java:245)
	at org.openrewrite.internal.ListUtils.map(ListUtils.java:269)
	at org.openrewrite.internal.InMemoryLargeSourceSet.edit(InMemoryLargeSourceSet.java:85)
	at org.openrewrite.scheduling.RecipeRunCycle.editSources(RecipeRunCycle.java:177)
	at org.openrewrite.RecipeScheduler.runRecipeCycles(RecipeScheduler.java:84)
	at org.openrewrite.RecipeScheduler.scheduleRun(RecipeScheduler.java:41)
	at org.openrewrite.Recipe.run(Recipe.java:436)
	at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:386)
	at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:132)
	at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:127)
	at com.spotify.bazel.rewrite.recipes.StaticImportsTest.staticImports(StaticImportsTest.java:21)
Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.Boolean.booleanValue()" because the return value of "org.openrewrite.java.style.SpacesStyle$BeforeLeftBrace.getClassLeftBrace()" is null
	at org.openrewrite.java.format.SpacesVisitor.visitClassDeclaration(SpacesVisitor.java:199)
	at org.openrewrite.java.format.SpacesVisitor.visitClassDeclaration(SpacesVisitor.java:31)
	at org.openrewrite.java.tree.J$ClassDeclaration.acceptJava(J.java:1370)
	at org.openrewrite.java.tree.J.accept(J.java:55)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:242)
	at org.openrewrite.java.format.SpacesVisitor.visit(SpacesVisitor.java:1130)
	at org.openrewrite.java.format.SpacesVisitor.visit(SpacesVisitor.java:31)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:154)
	at org.openrewrite.TreeVisitor.visitNonNull(TreeVisitor.java:173)
	at org.openrewrite.java.format.AutoFormatVisitor.visit(AutoFormatVisitor.java:87)
	at org.openrewrite.java.format.AutoFormatVisitor.visit(AutoFormatVisitor.java:43)
	at org.openrewrite.java.JavaVisitor.autoFormat(JavaVisitor.java:95)
	at org.openrewrite.java.AddImport.lambda$preVisit$3(AddImport.java:189)
	at org.openrewrite.internal.ListUtils.mapFirst(ListUtils.java:166)
	at org.openrewrite.internal.ListUtils.mapFirst(ListUtils.java:183)
	at org.openrewrite.java.AddImport.preVisit(AddImport.java:188)
	at org.openrewrite.java.AddImport.preVisit(AddImport.java:59)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:239)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:265)
	at org.openrewrite.TreeVisitor.visit(TreeVisitor.java:154)
	at org.openrewrite.Preconditions$Check.visit(Preconditions.java:159)
	at org.openrewrite.Preconditions$Check.visit(Preconditions.java:130)
	at org.openrewrite.scheduling.RecipeRunCycle.lambda$editSources$7(RecipeRunCycle.java:210)
	at org.openrewrite.table.RecipeRunStats$PhaseTimer.recordTimed(RecipeRunStats.java:142)
	at org.openrewrite.table.RecipeRunStats$RecipeTimers.recordEdit(RecipeRunStats.java:130)
	at org.openrewrite.table.RecipeRunStats.recordEdit(RecipeRunStats.java:58)
	at org.openrewrite.scheduling.RecipeRunCycle.lambda$editSources$8(RecipeRunCycle.java:206)
	at org.openrewrite.scheduling.RecipeStack.reduce(RecipeStack.java:60)
	at org.openrewrite.scheduling.RecipeRunCycle.lambda$editSources$9(RecipeRunCycle.java:179)
	at org.openrewrite.internal.InMemoryLargeSourceSet.lambda$edit$0(InMemoryLargeSourceSet.java:86)
	at org.openrewrite.internal.ListUtils.map(ListUtils.java:245)
	at org.openrewrite.internal.ListUtils.map(ListUtils.java:269)
	at org.openrewrite.internal.InMemoryLargeSourceSet.edit(InMemoryLargeSourceSet.java:85)
	at org.openrewrite.scheduling.RecipeRunCycle.editSources(RecipeRunCycle.java:177)
	at org.openrewrite.RecipeScheduler.runRecipeCycles(RecipeScheduler.java:84)
	at org.openrewrite.RecipeScheduler.scheduleRun(RecipeScheduler.java:41)
	at org.openrewrite.Recipe.run(Recipe.java:436)
	at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:386)
	at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:132)
	at org.openrewrite.test.RewriteTest.rewriteRun(RewriteTest.java:127)
	at com.spotify.bazel.rewrite.recipes.StaticImportsTest.staticImports(StaticImportsTest.java:21)
```

--- a/rewrite-java/src/main/resources/META-INF/rewrite/google-java-format.yml
+++ b/rewrite-java/src/main/resources/META-INF/rewrite/google-java-format.yml
@@ -28,4 +28,17 @@ styleConfigs:
   - org.openrewrite.java.style.SpacesStyle:
       # see https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
       beforeLeftBrace:
-        arrayInitializerLeftBrace: true
+        classLeftBrace: true
+        methodLeftBrace: true
+        ifLeftBrace: true
+        elseLeftBrace: true
+        forLeftBrace: true
+        whileLeftBrace: true
+        doLeftBrace: true
+        switchLeftBrace: true
+        tryLeftBrace: true
+        catchLeftBrace: true
+        finallyLeftBrace: true
+        synchronizedLeftBrace: true
+        arrayInitializerLeftBrace: true # Differs from IntelliJ default
+        annotationArrayInitializerLeftBrace: false

